### ISSUE TITLE
Deploy a mirror of the terraref betydb somewhere that the vice RStudio docker instances can read

### ISSUE THREAD

**Issue by Chris-Schnaufer** [terraref/tutorials#189]
Step 5
Split from #147

**Comment by Chris-Schnaufer** [comment]
@dlebauer Is this still an issue?

**Comment by dlebauer** [comment]
@Chris-Schnaufer yes, not sure how we want to handle it; I think this falls under the scope of @julianpistorius work

**Comment by Chris-Schnaufer** [comment]
There is a container available to automatically update the DB with TERRA REF data

**Comment by dlebauer** [comment]
dockerfile and docker-compose.yml required to deploy this (I think with automatic sync) is available here:

https://github.com/terraref/data-publication/tree/draft-internal-review/code/betydb_docker

See the README, ie. this should do it (credentials are in .pgpass):

```sh
docker-compose up -d postgres
docker-compose run --rm bety initialize
docker-compose run --rm bety sync
```
